Dockerfile

```diff
@@ -16,6 +17,7 @@ RUN poetry install --no-interaction --no-ansi --no-cache --only main
 FROM python:3.11-alpine as server
 
 RUN apk update && apk upgrade --no-cache libcrypto3 libssl3
+RUN apk add --no-cache alpine-sdk linux-headers
```

Comment:
Once the binaries are built these should no longer be needed.

Reply:
Fixed